src/libraries/System.Linq.Expressions/tests/DynamicExpression/DynamicExpressionTests.cs

```diff
@@ -75,6 +75,7 @@ public void TypeProperty(int size, Type type)
         }
 
         [Theory, MemberData(nameof(SizesAndTypes))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/69985", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
Do we need an ActiveIssue for this? This should fix the immediate issue:

```xml
    <Assembly Name="System.Linq.Expressions">
      <Type Name="System.Runtime.CompilerServices.CallSite`1" Dynamic="Required All" />
    </Assembly>
```

CallSite is constrained to be a class.